src/models/network/NetworkBaseController.js

```diff
@@ -83,7 +84,7 @@ export default class NetworkBaseController {
   }
 
   async uploadContract(contractAlias, contractName) {
-    const contractClass = Contracts.getFromLocal(contractName);
+    const contractClass = Contracts._getFromBuildDir(contractBuildDirectory(), contractName);
```

Comment:
i understand we shouldn't be calling `_getFromBuildDir` as that's meant to be a private function, but perhaps this can lead to a proposed change in `zos-lib` as well to make that `getFromBuildDir` public function

Reply:
Actually, if we can get this merged in first, then that would solve this issue as well:

https://github.com/zeppelinos/zos-lib/pull/183